### ISSUE TITLE
Soundflower folder in Applications folder can't be accessed - no permissions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mattscotthudson/soundflower#68]
```
What steps will reproduce the problem?
1. Un-installed Soundflower, restarted.
2. Reinstalled Soundflower, restarted machine
3. Attempt to double-click on Soundflower folder to open but it has a red minus 
sign on it.  No 
permissions.

What is the expected output? What do you see instead?
I should be able to go into the Soundflower folder to launch Soundflower bed.  
There is a red 
minus sign on the folder indicating I do not have permissions.  I removed the 
application using 
your uninstaller, restarted, reinstalled a new version restarted but it still 
has the minus sign on 
the folder and I am unable to access the folder

What version of the product are you using? On what operating system?
1.5.2 (although I was using 1.5.1 and it had the same issue).  Using Mac OS X 
10.6.3 with Intel 
Macbook Pro 2.33 ghz.


Please provide any additional information below.
Soundflower inputs and outputs do show up but I can't access the folder to 
launch Soundflower 
bed.  For some reason the folder in the Application area is created without 
permissions for me to 
access.


```

Original issue reported on code.google.com by `joe.rizz...@gmail.com` on 11 May 2010 at 5:19

**Comment by GoogleCodeExporter** [comment]
```
Sorry, this seems to be the same as Issue #42 that was previously reported.  
The permissions set for 'everyone' were set to 'none' even on a clean 
installation.  I had to use 'get info' on the 
application icon, click the lock icon, log in as my administrator account and 
change 'Everyone' permission to 
'read only' to be able to access the folder.
```

Original comment by `joe.rizz...@gmail.com` on 11 May 2010 at 5:48

**Comment by GoogleCodeExporter** [comment]
```
By default, the installer installs the application directory owned by UID 501. 
On my system, my account is UID is 502. Run the following to fix:

sudo chown -R `id -u`:wheel /Applications/Soundflower/
```

Original comment by `rob...@gmail.com` on 20 Jun 2010 at 10:43

**Comment by GoogleCodeExporter** [comment]
```
Thanks -- will try to get this into the next installer.

http://github.com/tap/Soundflower/issues#issue/8

```

Original comment by `74obje...@gmail.com` on 15 Oct 2010 at 6:23
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in the just-released version 1.6 release candidate.
```

Original comment by `t...@electrotap.com` on 3 Feb 2012 at 8:48
- Changed state: **Fixed**
